src/vm/i386/unixstubs.cpp

```diff
@@ -41,7 +41,7 @@ extern "C"
         PORTABILITY_ASSERT("Implement for PAL");
     }
 
-    DWORD getcpuid(DWORD arg, unsigned char result[16])
+    DWORD STDCALL getcpuid(DWORD arg, unsigned char result[16])
     {
```

Comment:
How about __stdcall instead of STDCALL? cgensys.h declares this function as ``extern "C" DWORD __stdcall getcpuid(DWORD arg, unsigned char result[16]);``